features/layout/footer-navigation/footer-navigation.module.scss

```diff
@@ -0,0 +1,98 @@
+@use "@styles/color";
+@use "@styles/font";
+@use "@styles/space";
+@use "@styles/breakpoint";
+@use "@styles/misc";
+
+.footer {
+  background-color: color.$gray-50;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: space.$s6;
+  padding: space.$s6;
+
+  @media (min-width: breakpoint.$desktop) {
+    flex-direction: row;
+    justify-content: space-between;
+    height: space.$s15;
+    padding: 0 space.$s6;
+  }
+}
+
+.anchorContainer {
+  display: flex;
+  gap: space.$s6;
+
+  a {
+    color: color.$gray-500;
+    text-decoration: none;
+  }
+}
+
+.logoContainer {
+  @media (min-width: breakpoint.$desktop) {
+    width: 33%;
```

Comment:
Using `flex: 1` would be a bit more flexible here instead of setting the width I think.